pyproject.toml

```
@@ -0,0 +1,75 @@
+[project]
+name = "remote-server"
+version = "0.1.0"
+description = "NVDA Remote Access remote relay server."
+readme = "README.md"
+requires-python = ">=3.13.3"
```

I think we can probably get away with always using the latest 3.13

```suggestion
requires-python = "~=3.13"
```